student.rb

```diff
@@ -1,9 +1,16 @@
 require_relative 'person'
 
 class Student < Person
-  def initialize(classroom, age, name = 'unknown', parent_permission: true)
-    super(name, age, parent_permission)
+  attr_reader :classroom
+
+  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
+    @classroom = classroom
+  end
+
+  def classroom=(classroom)
     @classroom = classroom
+    classroom.students.push(self)
   end
```

Comment:
- [ ] You should add a condition that verifies if the "classroom.students" already have the student:
```ruby
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```

Reply:
Ok, noted with thanks.